### DISCUSSION TITLE
All pals show as ghosted

### DISCUSSION THREAD

**Discussion by joe111487** [KrisCris/Palworld-Pal-Editor#50] [Q&A]
Once I load my save is shows all my pals as ghosted, I can still edit the level of them but that's about it. It saves and upon loading into the game the levels are what I edited them to but that's the only one that sticks. I'm about lost.

**Comment by KrisCris** [comment]
You are not running the latest game. They have changed the save format, showing them as ghost pals is so far the backward compatibility we can get.
